R/render.R

```diff
@@ -32,38 +35,59 @@ create_prevelance_renderer <- function(
         parameters
       )
       asymptomatic_detected <- bitset_at(asymptomatic, bernoulli_multi_p(prob))
-      
-    } else if (parameters$parasite == "vivax"){
-      # The vivax model defines asymptomatic infections as being detectable by
-      # light microscopy
-      prob <- rep(1, asymptomatic$size())
-      asymptomatic_detected <- state$get_index_of('A')
+    } else {
+      asymptomatic_detected <- asymptomatic
     }
     
     clinically_detected <- state$get_index_of(c('Tr', 'D'))
-    detected <- clinically_detected$copy()$or(asymptomatic_detected)
-
+    lm_detected <- detected <- clinically_detected$copy()$or(asymptomatic_detected)
```

Comment:
This may be something that bugs me an not anyone else, but can we keep these as two seperate assignments please.

Reply:
Ooops - this was actually a mistake (shouldn't have the middle part). I'll fix that.